devops/rpmbuild/extrafiles/basecoin/usr/share/basecoin/key2.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This is an example for tutorials. I didn't want to change the name (although I agree, test-key-2.json would be less confusing) because this is how it is mentioned in the tutorials.
The key file (priv_validator.json) is generated dynamically. :)